src/CMakeLists.txt

```diff
@@ -702,6 +708,11 @@ if(TINYSPLINE_FLOAT_PRECISION)
   list(APPEND TINYSPLINE_CXX_DEFINITIONS "TINYSPLINE_FLOAT_PRECISION")
   list(APPEND TINYSPLINE_BINDING_CXX_DEFINITIONS "TINYSPLINE_FLOAT_PRECISION")
 endif()
+if(NOT TINYSPLINE_SERIALIZATION)
+  list(APPEND TINYSPLINE_C_DEFINITIONS "TINYSPLINE_NO_SERIALIZATION")
+  list(APPEND TINYSPLINE_CXX_DEFINITIONS "TINYSPLINE_NO_SERIALIZATION")
+  list(APPEND TINYSPLINE_BINDING_CXX_DEFINITIONS "TINYSPLINE_NO_SERIALIZATION")
```

Comment:
We will probably never build one or more bindings without serialization support.